common.mk

```diff
@@ -27,6 +27,12 @@ $(warning Looks like TerraForm is not installed. This is ok as long as you're no
           Deploying new lambdas is still possible with `make deploy` but `make terraform` will not work.)
 endif
 
+ifneq ($(shell python -c "import wheel as w; \
+						  from pkg_resources import parse_version as p; \
```

Comment:
Indent with spaces and align such that `from` is under `import`. Same with next line.